website/docs/r/monitor.html.markdown

```diff
@@ -20,10 +20,7 @@ resource "sumologic_monitor" "tf_logs_monitor_1" {
   content_type = "Monitor"
   monitor_type = "Logs"
   evaluation_delay = "5m"
-  queries {
-    row_id = "A"
-    query  = "_sourceCategory=event-action info"
-  }
```

Comment:
Any specific reason for removing `queries`? It will be needed for this monitor type.